### PR TITLE
configure in qucs

### PR DESCRIPTION
**Pull request by felix-salfelder**
- Romano Giannetti found a whitespace issue and proposed a fix.
- list pkg-config as a dependency.

### CODE REVIEW

File: qucs/README [hunk 1]
```diff
@@ -36,6 +36,8 @@ Requirements
 
 Qucs needs Qt(R) by Digia(R) <http://www.digia.com>.
```
Comment (in3otd) [inline qucs/README]: not related but Qt development is now done by the Qt Group Plc, https://www.qt.io :grin:

**Comment by guitorri** [comment]
No need to update qucs-test.

**Comment by felix-salfelder** [comment]
On Wed, Jul 25, 2018 at 11:58:32AM -0700, Guilherme Brondani Torri wrote:
> No need to update qucs-test.

thanks, fixed.

(still don't understand how this always happens since recently. i never
touched any of this...)

File: qucs/configure.ac [hunk 2]
```diff
@@ -236,7 +236,7 @@ case $host_os in
 
     dnl Enable C++11 support
     CXXFLAGS="$CXXFLAGS -std=c++0x"
-    PKG_CHECK_MODULES([QTest], [${qt_name}Test ])
+    PKG_CHECK_MODULES([QTest], [${qt_name}Test])
```
Comment (in3otd) [inline qucs/configure.ac]: by the way, any idea why this works with and without the space on my system and on Travis too?
Here everything also works fine also if I remove this line, so in which case is it needed??

Trying to understand what `PKG_CHECK_MODULES` does, I saw that "[this guy](http://tirania.org/blog/archive/2012/Oct-20.html)" and others discourage its usage; FWIW, I knowing nothing about it.

**Comment by guitorri** [comment]
If you don' t `git update submodule` and `git add -a` you also pick up the old `qucs-test` hash.

The QtTest is needed if you run `make check`.

**Comment by felix-salfelder** [comment]
On Wed, Jul 25, 2018 at 07:35:42PM +0000, in3otd wrote:
>      dnl Enable C++11 support
>      CXXFLAGS="$CXXFLAGS -std=c++0x"
> -    PKG_CHECK_MODULES([QTest], [${qt_name}Test ])
> +    PKG_CHECK_MODULES([QTest], [${qt_name}Test])
> 
> by the way, any idea why this works with and without the space on my
> system and on Travis too?

don't know, perhaps it's actually a bug in an older pkg.m4. I wouldn't
have put the space in, if it had not worked for me..

> Trying to understand what `PKG_CHECK_MODULES` does, I saw that "[this
> guy](http://tirania.org/blog/archive/2012/Oct-20.html)" and others
> discourage its usage; FWIW, I knowing nothing about it.

interesting article, I never gave it a thought. otoh, if
PKG_CHECK_MODULES only induces trouble once every 10+ years, and only due
to stupid whitespace, then why bother? it's just as easy to get a shell
script wrong in a subtle way, YMMV.

**Pull request merged**